docs/herramientas_test.md

```diff
@@ -0,0 +1,91 @@
+## Criteri di Selezione
+Per il nostro progetto, abbiamo preso in considerazione i seguenti criteri:
+
+1. **Evitazione del debito tecnico**: È essenziale che gli strumenti permettano di scrivere test di alta qualità, evitando la duplicazione del codice e facilitando l'integrazione continua.
+2. **Raccomandazioni da parte di SNYK**: Gli strumenti scelti devono essere consigliati da parte di SNYK e avere una buona reputazione per stabilità e prestazioni.
+
+---
+
+## 1. **Scelta del Test Runner**
```

Comment:
Magari è meglio cominciare con l'asserzioni, dato che essi sono usati per gli test runners?

Reply:
Ha ragione, può essere più logico presentare prima la scelta delle asserzioni e solo in seguito il Test Runner